backend/src/index.js

```diff
@@ -0,0 +1,41 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+import express from 'express';
+import cors from 'cors';
+import helmet from 'helmet';
+
+import routes from './routes';
+import database from './config/database';
+import {
+  appErrorHandler,
+  genericErrorHandler,
+  notFound
+} from './middlewares/error.middleware';
+import logger, { logStream } from './config/logger';
+
+import morgan from 'morgan';
+
+const app = express();
+const host = process.env.APP_HOST;
+const port = process.env.APP_PORT;
+const api_version = process.env.API_VERSION;
+
+app.use(cors());
+app.use(helmet());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+app.use(morgan('combined', { stream: logStream }));
+
+database();
```

Comment:
@ManishKumar0007 
start the DB connection after the server starts listening on the configured port.

Reply:
done